src/OSPSuite.UI/Controls/UxChartControl.cs

```diff
@@ -37,12 +37,11 @@ public UxChartControl(bool useDefaultPopupMechanism = true, bool addCopyToClipbo
          Titles.Add(_title);
          Titles.Add(_description);
 
-         _clipboardTask = new ClipboardTask();
          _barManager = new BarManager {Form = this};
          PopupMenu = new PopupMenu(_barManager);
 
          if (useDefaultPopupMechanism)
-            initializePopup(addCopyToClipboardMenu);
```

Comment:
no default copy to clipboard anymore

Reply:
>no default copy to clipboard anymore

Is that because Copy2Clipboard is now only available in preview-mode?

Reply:
no it's because we should never bypass the copytoclipboard accidentally. With this flag removed every plot control used will always use the watermark if needed. no way araound that